app/src/main/java/com/asfoundation/wallet/ui/SettingsFragment.kt

```diff
@@ -38,30 +50,100 @@ class SettingsFragment : PreferenceFragmentCompat() {
 
   override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
     super.onViewCreated(view, savedInstanceState)
+    presenter.present()
+  }
+
+  override fun onResume() {
+    presenter.present()
+    super.onResume()
+  }
+
+  override fun onDestroyView() {
+    presenter.stop()
+    super.onDestroyView()
+  }
+
+  private fun startBrowserActivity(uri: Uri, newTaskFlag: Boolean) {
+    try {
+      val intent = Intent(Intent.ACTION_VIEW, uri)
+      if (newTaskFlag) {
+        intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
+      }
+      startActivity(intent)
+    } catch (exception: ActivityNotFoundException) {
+      exception.printStackTrace()
+      if (view != null) {
```

Comment:
This is redundant ?. only calls/execute the following if it's non null.
Replace with `view?.let {
          Snackbar.make(it, R.string.unknown_error, Snackbar.LENGTH_SHORT)
              .show()
        }`